cukedoctor-section-layout/src/main/java/com/github/cukedoctor/sectionlayout/BasicSection.java

```diff
@@ -175,4 +176,14 @@ private void renderChildren(CukedoctorDocumentBuilder docBuilder, I18nLoader i18
     public int getOrder() {
         return order;
     }
+
+    @Override
+    public Iterable<Feature> getFeatures() {
+        Iterable<Feature> aggregate = hasRoot() ? root.getFeatures() : Collections.emptyList();
+        for (Section section : getChildren()) {
+            aggregate = Iterables.concat(aggregate, section.getFeatures());
```

Comment:
Can we replace guava with Stream.concat or flatMap + a collector to combine?
I would add guava dependency as last resource because It can add classpath issues

Reply:
> Can we replace guava with Stream.concat or flatMap + a collector to combine?
> I would add guava dependency as last resource because It can add classpath issues

Will do

Reply:
Done in 042aaf9